helfi_features/helfi_profile_block/templates/profile-block.html.twig

```diff
@@ -0,0 +1,44 @@
+{% if logged_in %}
```

Comment:
This file should be in HDBT, not in platform-config

Reply:
Good catch this one! Forgot all about it. Moved it to hdbt.